osu.Game/Overlays/BeatmapListingOverlay.cs

```diff
@@ -176,6 +176,8 @@ private void onSearchFinished(BeatmapListingFilterControl.SearchResult searchRes
             }
             else
             {
+                newCards = newCards.Except(foundContent);
```

Comment:
I'm okay with merging this in as the effort has already been spent on making this work / adding testing, but if it is to be done, this needs an inline comment mentioning this is a temporary hotfix, and linking to the osu-web issue thread.

Reply:
I already attached one in the test case, but I guess I could add one here.